frontend/src/core/comments/components/Comment.js

```diff
@@ -58,30 +67,19 @@ export default function Comment(props: Props) {
                     date={ new Date(comment.dateIso) }
                     title={ `${comment.createdAt} UTC` }
                 />
-                { '\u2022' }
-                <Localized
-                id='comments-Comment--delete-button'
-                attrs={{ title: true }}
-                >
-                    <button
-                        title='Delete'
-                        onClick={ deleteComment }
-                    >
-                        DELETE
-                    </button>
-                </Localized>
-                { '\u2022' }
-                <Localized
-                id='comments-Comment--share-button'
-                attrs={{ title: true }}
-                >
-                    <button
-                        title='Share'
-                        onClick={ shareComment }
+                { !canDelete ? null :
+                    <Localized
+                        id='comments-Comment--delete-button'
+                        attrs={{ title: true }}
                     >
-                        SHARE
-                    </button>
-                </Localized>
+                        <button
+                            title='Delete'
```

Comment:
The title here is useless because it's just a repeat of the button label.

Reply:
Does it need the title for Localization though?